cocotb/bfms.py

```diff
@@ -0,0 +1,206 @@
+###############################################################################
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+###############################################################################
+
+import os
+import importlib
```

Comment:
This is new in python 3.1 - are you waiting for #767 to be resolved?

Reply:
It might make things simpler. What's the timeline for resolving #767? I'd hate to leave this PR in limbo, but if #767 is close to being resolved I can wait.

Reply:
It's already been resolved, so this is no longer an issue :)